dialog/romimportsystem.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
+
+_CONFIG=/opt/retronas/config/retronas.cfg
+source $_CONFIG
+source ${LIBDIR}/common.sh
+cd ${DIDIR}
+
+## If this is run as root, switch to our RetroNAS user
+DROP_ROOT
+CLEAR
+
+rn_import_system() {
+
+  local MENU_NAME=romimportsystem
+  READ_MENU_JSON "${MENU_NAME}"
+  READ_MENU_TDESC "${MENU_NAME}"
+  DLG_MENUJ "${MENU_TNAME}" 10 "${MENU_BLURB}"
+  
+
+    while read -r line; do
+        lineArr=($line)
+        STR=${lineArr[0]}
+        SUB="id"
+        if [[ "$STR" == *"$SUB"* ]]; then
+            SYSTEMS+=${lineArr[1]}' '
+        fi
+    done < <(/opt/retronas/scripts/romimport.sh -l)
+
+    i=2
+    for s in $SYSTEMS:
+    do
+        SYSTEM_ARR[$i]=$s
+        ((++i))
+    done
+
+  while true
+  do
+    source $_CONFIG
+    
+    case ${CHOICE} in
+      01)
+        EXIT_OK
+      ;;
+      02)
+        CLEAR
+        /opt/retronas/scripts/romimport.sh -t ${SYSTEM_ARR[2]}
```

Comment:
This should be able to be moved to a function to reduce duplication throughout the file and call it with the menu entry param